packages/client/modules/userDashboard/containers/UserColumns/UserColumnsContainer.tsx

```diff
@@ -24,33 +30,51 @@ const UserColumnsContainer = (props: Props) => {
       : nodes
 
     const teamFilteredNodes = teamFilterId
-      ? dashSearchNodes.filter((node) => node.team.id === teamFilterId)
+      ? dashSearchNodes.filter((node) => node.teamId === teamFilterId)
       : dashSearchNodes
 
-    return teamFilteredNodes.map((node) => ({
+    const teamMemberFilteredNodes = userFilterId
+      ? teamFilteredNodes.filter((node) => {
+        return node.userId === userFilterId
+      })
+      : teamFilteredNodes
+
+    return teamMemberFilteredNodes.map((node) => ({
       ...node
     }))
-  }, [teamFilterId, tasks, dashSearch])
+  }, [teamFilterId, userFilterId, tasks, dashSearch])
   {
     const {
       viewer: {teams}
     } = props
 
-    return <TaskColumns area={AreaEnum.userDash} tasks={filteredTasks} teams={teams} />
+    // iff 1 user is selected, we show team names at the footer; otherwise we show task owner name
+    const areaForTaskCard = userIds && userIds.length === 1 ? AreaEnum.userDash : AreaEnum.teamDash
+    const filteredTeams = userIds ? teams.filter(({teamMembers, id: teamId}) => {
+      const inTeam = teamMembers.find(({userId}) => userIds.includes(userId)) != undefined
```

Comment:
interesting that the linter didn't pick this up, we may have to revisit those rules.
`!=` works here for `undefined` and `null`, but in general, it's a pretty big footgun, so the JS community as a whole has moved away from it & only uses strict unequals `!==`. in this case, i'd probably just prefix with `!!`, but that's personal preference